Libraries/Opc.Ua.Client/Session/SessionAsync.cs

```diff
@@ -1051,6 +1061,300 @@ List<ServiceResult> errors
         }
         #endregion
 
+        #region Combined Browse/BrowseNext
+        /// <inheritdoc/>
+        public async Task<(
+            List<ReferenceDescriptionCollection>,
+            IList<ServiceResult>
+            )>
+                ManagedBrowseAsync(
+                    RequestHeader requestHeader,
+                    ViewDescription view,
+                    IList<NodeId> nodesToBrowse,
+                    uint maxResultsToReturn,
+                    BrowseDirection browseDirection,
+                    NodeId referenceTypeId,
+                    bool includeSubtypes,
+                    uint nodeClassMask,
+                    bool executeDefensively = false,
+                    CancellationToken ct = default
+            )
+        {
+            var result = new List<ReferenceDescriptionCollection>();
```

Comment:
set the known list size in the constructor, saves some alloc/copy

Reply:
done.